installation-bundle/src/InstallTool.php

```diff
@@ -227,6 +232,16 @@ public function hasConfigurationError(array &$context): bool
 
                 return true;
             }
+        } else {
+            // Use default engine if none is configured
+            $statement = $this->connection->query('SHOW ENGINES');
```

Comment:
I think `SELECT @@default_storage_engine` or `SHOW VARIABLES WHERE variable_name = 'default_storage_engine'` might be simpler here?

Reply:
Good point, I updated the pull request